### ISSUE TITLE
Empty hyperlinks with a title do not cause a failure to be reported

### ISSUE THREAD

**Issue by davidsauntson** [squizlabs/HTML_CodeSniffer#292]
Hyperlinks with no content or child nodes but with a meaningful `title` attribute do not pass [WCAG2.1 2.4.4](https://www.w3.org/TR/WCAG21/#link-purpose-in-context), since the `title` attribute alone does not constitute a sufficient method for complying with the standard.  

[Technique H33](https://www.w3.org/TR/WCAG20-TECHS/H33) talks about supplementing with a title rather than relying on a title alone.

Testing a link like this: `<a href="https://www.github.com" title="Visit GitHub"></a>` with HTML_CodeSniffer does not report a failure and it should. 

Other testing tools like WAVE do report this as a failure.
